{{cookiecutter.project_slug}}/roadmap/ignite_roadmap.md

```diff
@@ -0,0 +1,64 @@
+# {{ cookiecutter.project_name }} Roadmap
+
+This document lists general directions that core team is interested to see developed in {{ cookiecutter.project_name }}
+
+## Principal goals
+
+<<DESCRIBE THE PRINCIPALS GOALS OF THE PROJECT?>> For example:
+
+<<continue maintaining high-quality, well-tested and documented modules>>
```

Comment:
sorry, maybe I was not clear about that
instead of all this items for each section, each section would have just one, maybe something like
`- <<add your item here>>`

Reply:
each item would change from project to project .. 
the items you provided was specific for ignite

Reply:
@xmnlab Ok, sorry. I didn't understand you. I left it like this because as I put: For example.
But I modify it. Thanks